src/main/resources/edu/cornell/kfs/module/ld/businessobject/datadictionary/PositionData.xml

```diff
@@ -57,45 +57,46 @@
   </bean>
       
   <!-- Business Object Inquiry Definition -->
+  <bean id="PositionData-inquiryDefinition-override" parent="DataDictionaryBeanOverride">
+    <property name="beanName" value="PositionData-inquiryDefinition" />
+    <property name="fieldOverrides">
+      <list>
+          <!-- Bean override for new standalone page field presentation -->
+          <bean parent="FieldOverrideForListElementInsert"
+            p:propertyName="sections[0].fields"
+            p:propertyNameForElementCompare="name"
+            p:element-ref="PositionData-departmentId"
+            p:insertAfter-ref="PositionData-extension-orgCode"/>
 
-  <bean parent="DataDictionaryBeanOverride">
-      <property name="beanName" value="PositionData-inquiryDefinition" />
-      <property name="fieldOverrides">
-          <list>
-              <bean parent="FieldOverrideForListElementInsert">
-                  <property name="propertyName" value="inquirySections[0].inquiryFields" />
-                  <property name="propertyNameForElementCompare" value="attributeName" />
-                  <property name="element">
-                      <bean parent="FieldDefinition" p:attributeName="departmentId" />
-                  </property>
-                  <property name="insertAfter">
-                      <list>
-                          <bean parent="FieldDefinition" p:attributeName="extension.orgCode" />
-                      </list>
-                  </property>
-              </bean>
-          </list>
-      </property>
-  </bean>
-  
-  <bean parent="DataDictionaryBeanOverride">
-      <property name="beanName" value="PositionData-inquiryDefinition" />
-      <property name="fieldOverrides">
-          <list>
-              <bean parent="FieldOverrideForListElementInsert">
-                  <property name="propertyName" value="inquirySections[0].inquiryFields" />
-                  <property name="propertyNameForElementCompare" value="attributeName" />
-                  <property name="element">
-                      <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate" />
-                  </property>
-                  <property name="insertAfter">
-                      <list>
-                          <bean parent="FieldDefinition" p:attributeName="extension.inactivationDate" />
-                      </list>
-                  </property>
-              </bean>
-          </list>
-      </property>
+          <bean parent="FieldOverrideForListElementInsert"
+            p:propertyName="sections[0].fields"
+            p:propertyNameForElementCompare="name"
+            p:insertAfter-ref="PositionData-extension-inactivationDate"/>
```

Comment:
I realize that the field-override bean's "insertAfter" property allows for single objects or lists of objects, but since you have overrides for similar sections that have a mix-and-match of using the object way or the list way, I would suggest updating the 4 overrides to use the same strategy for consistency.

Also, if it's not too much trouble, it would be helpful to change the line formatting on these sections to be consistent with our typical formatting.